pkg/logs/logs.go

```diff
@@ -29,17 +35,43 @@ const (
 	DebugLvl = 1
 )
 
+// InitDevelLoggers Configure zap backend development logger
+func InitDevelLoggers() {
+	InitLoggers(true, "", "", "", "rfc3339")
+}
+
 // InitLoggers Configure zap backend for controller-runtime logger.
-func InitLoggers(development bool, fs *flag.FlagSet) {
+func InitLoggers(development bool, encoder string, logLevel string, stackTraceLevel string, timeEncoding string) {
+
+	flagSet := flag.NewFlagSet("zap", flag.ContinueOnError)
+
+	opts := crzap.Options{ZapOpts: []zap.Option{zap.WithCaller(true), zap.AddCallerSkip(-1)}}
+	opts.BindFlags(flagSet)
+	_ = flagSet.Set("zap-devel", strconv.FormatBool(development))
+	if len(encoder) > 0 {
+		_ = flagSet.Set("zap-encoder", encoder)
```

Comment:
Shouldn't we panic with some kind of output on `stderr` explaining the configuration error here?